src/olympia/devhub/templates/devhub/addons/edit.html

```diff
@@ -25,6 +25,17 @@ <h2>{{ addon.name }}</h2>
 
 <section class="primary" role="main">
 <div id="edit-addon" class="devhub-form">
+  {% if addon.current_version and addon.current_version.previews %}
+    <h3>
+      {{ _('Preview') }}
+    </h3>
+    <div class="edit-addon-section">
+      {% for preview in addon.current_version.previews.all() %}
+        <img src="{{ preview.image_url }}" title="{{ _('Preview of listed version') }}"
+            alt="[{{ _('Preview still being generated - come back later or refresh the page.') }}]">
```

Comment:
Not sure about the `[` and `]` but that's just a nit.

Reply:
I just wanted some way of differentiating it.  Could possibly have been done with css but meh.  Patches welcome.